### PR TITLE
Documentation website improvements

### PR DESCRIPTION
**Pull request by timocov**
**Type of PR:** docs enhancements

**PR checklist:**

- [ ] Addresses an existing issue: no
- [ ] Includes tests: no
- [x] Documentation update

### CODE REVIEW

File: website/docusaurus.config.js [hunk 1]
```diff
@@ -135,6 +135,9 @@ const config = {
 				entryPoints: ['../dist/typings.d.ts'],
 				readme: 'none',
 				disableSources: true,
+				watch: true,
+				preserveWatchOutput: true,
```
Comment (edew) [inline website/docusaurus.config.js:139]: I'm not sure we should hardcode watch mode? How would this affect running the build on CI?

if you want to enable it locally we can use env vars https://github.com/tgreyuk/typedoc-plugin-markdown/tree/master/packages/docusaurus-plugin-typedoc#watch-mode

```
"start": "TYPEDOC_WATCH=true docusaurus start",
"build": "TYPEDOC_WATCH=false docusaurus build",
```

Comment (timocov) [reply website/docusaurus.config.js:139]: > How would this affect running the build on CI?

I tried to run `npm run build` locally it worked 🤷

Comment (timocov) [reply website/docusaurus.config.js:139]:
```
$ npm run build

> @ build D:\projects\lightweight-charts\website
> docusaurus build


[en] Creating an optimized production build...
Warning: You are running with an unsupported TypeScript version! TypeDoc supports 4.0, 4.1, 4.2, 4.3, 4.4
Info: Starting compilation in watch mode...
Info: Found 0 errors. Watching for file changes.
Info: Documentation generated at ./docs/api

✔ Client


✔ Server
  Compiled successfully in 25.18s


✔ Client


● Server █████████████████████████ cache (99%) shutdown IdleFileCachePlugin
 stored

Success! Generated static files in "build".

Use `npm run serve` command to test your build locally.
```

Comment (timocov) [reply website/docusaurus.config.js:139]: I mean, it doesn't wait for changes and just finishes the process so it seems that it works correctly.

**Review by edew — APPROVED** [review]

**Pull request merged**